app/src/main/java/com/example/myapplication/SecondFragment.java

```diff
@@ -24,19 +24,7 @@
 public class SecondFragment extends Fragment {
 
     private FragmentSecondBinding binding;
-    public static int amp = 0;
-    public static int speakerUnamp = 0;
-    public static int speakerAmp = 0;
-    public static boolean broke = false;
-    public static boolean defense = false;
-    public enum Intake{
-        NONE,
-        GROUND,
-        SOURCE,
-        BOTH
-
-    }
-    public static SecondFragment.Intake intakeMethod = SecondFragment.Intake.NONE;
+    MatchData.NoteAcquisition intakeMethod = MatchData.NoteAcquisition.NONE;
```

Comment:
This doesn't need to be an instance field; it can be a local variable inside `onItemSelected` in the intakeMethod handler.